index.html

```diff
@@ -20,14 +20,36 @@
           width="20px"
         />
         <div id="myDropdown">
-          <a href="">Portoflio</a>
-          <a href="">About</a>
-          <a href="">Contact</a>
+          <a href="#works_section">Portoflio</a>
+          <a href="#about_me">About</a>
+          <a href="#contact">Contact</a>
           <a href=""
             ><img src="./images/Icon - Mail.png" alt="email-link" width="15"
           /></a>
         </div>
       </nav>
+      <div id="mobile-menu-modal" class="toogleModalDisplay">
+        <div id="mobile-menu">
+          <div id="modal-close-icon">
+            <img
+              src="./images/modal-close-icon.png"
+              alt="close-mobile-menu"
+              width="20rem"
+            />
```

Comment:
- [ ] OPTIONAL
 Kindly ensure that the _hamburger_ and the _close_ button are in the same position when toggled.

- [ ] OPTIONAL
To improve the **user experience** it will be good practice to ensure that the _hamburger_ and the _close button_ behave as buttons when the mouse hovers over them. We can improve upon this aspect by adding the CSS attribute `cursor: pointer` to the _hamburger_ and the _close button_ on _mouse hover_ events.